src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -95,7 +96,24 @@ public void tearDown() throws Exception {
 
   // tikv-4.0 does not support atomic api
   @Ignore
```

Comment:
We can enable this test now, by
- set default test tikv version from `relese-4.0` to `release-5.0`
- add tikv config: 
```
[storage]
enable-ttl = true
```